### PR TITLE
configure.ac: update glib min. version dependency

### PR DESCRIPTION
**Pull request by juliusmilan**
libreport until now required glib 2.21, g_list_free_full (which caused the troubles) is already present in glib version 2.28, but I updated requirement to 2.43 because abrt requires that one (2.43) and it is still old enough.
Tried to build it and update, that worked fine but unfortunately I didn't find a way to test it exactly as described in https://bugzilla.redhat.com/show_bug.cgi?id=1445669, i.e. during package update.

Related to rhbz#1445669

### CODE REVIEW

**Comment by jfilak** [comment]
> g_list_free_full (which caused the troubles)

What troubles are you talking about? I have no access to the Bugzilla bug.

**Comment by juliusmilan** [comment]
There is dependency on glib 2.21 in libreport, however g_list_free_full is used by libreport and it is in glib just since 2.28. When user has glib of pre-2.28 version, libreport of version dependent on pre-2.28 glib and tries to update libreport, it doesn't update glib and complains, that doesn't know g_list_free_full.

Recently I got the suggestion, that it is needed to specify the version in spec file too and also some tips how to test it, will update this soon.

**Comment by juliusmilan** [comment]
Added build and also install time requirement into spec file.
Tested. Update of libreport (using rpm -Uvh ...) when just older version of glib is present doesn't proceed anymore.

File: libreport.spec.in [hunk 1]
```diff
@@ -54,6 +57,7 @@ BuildRequires: xz
 BuildRequires: lz4
 Requires: libreport-filesystem = %{version}-%{release}
 Requires: satyr >= 0.18
+Requires: glib2 >= %{glib_ver}
```
Comment (jfilak) [inline libreport.spec.in]: Are you sure this explicit library requires is needed? rpmbuild should be able to add the dependency automatically:
https://fedoraproject.org/wiki/Packaging:Guidelines#Explicit_Requires

Comment (jfilak) [inline libreport.spec.in]: Run rpmlint on the resulting rpm packages:
https://linux.die.net/man/1/rpmlint

Comment (xsuchy) [inline libreport.spec.in]: @jfilak rpmbuild automatically add just libglib-2.0.so.0()(64bit) without the version. And as stated there are unfortunatelly older version of glib2 which are not compatible with libreport.

Comment (jfilak) [inline libreport.spec.in]: Ha, I thought GLib2 uses also the minor version in its soname.

**Pull request merged**